fubectl.source

```diff
@@ -164,6 +164,14 @@ klog() {
     kubectl logs -n ${arg_pair} -c "${container_choosen}" --tail="${line_count}" "$@"
 }
 
+# [kkonfig] select a file in current directory and set it as $KUBECONFIG
+kkonfig() {
+    local kubeconfig
+    kubeconfig=$(_inline_fzf_nh) &&
+    export KUBECONFIG=$PWD/$kubeconfig &&
+    exec $SHELL
```

Comment:
Why are there `&&` statements in here?

Reply:
To avoid setting $KUBECONFIG and exec-ing into $SHELL, if file selection is canceled. FZF will return non 0 exit code in case when I hit ctrl-c instead of enter when selecting a file.

Reply:
So we do not use `set -e`? I don't see the necessesity for the `&&` in any other function (except for when it's used for a "short if" construct).

Reply:
Of course we don't use `set -e` since this script will be sourced in the user's shell.

Reply:
Here's what's happening with `set -x / set +x` inside the kkonfig.

I've selected the file:
```
$ kkonfig
+ local kubeconfig
++ fzf --multi --ansi -i -1 --height=50% --reverse -0 --inline-info --border
+ kubeconfig=kubermatic.support
+ export KUBECONFIG=/Users/kron/DEV/loodse/vault/kubermatic.support
+ KUBECONFIG=/Users/kron/DEV/loodse/vault/kubermatic.support
+ exec /usr/local/bin/bash
```

I've canceled selection:
```
$ kkonfig
+ local kubeconfig
++ fzf --multi --ansi -i -1 --height=50% --reverse -0 --inline-info --border
+ kubeconfig=
+ set +x
```

Reply:
I've changed it to `kubeconfig=$(_inline_fzf_nh) || return`.